pkg/template/config_context.go

```diff
@@ -320,7 +321,15 @@ func (ctx ConfigCtx) localImageName(imageRef string) string {
 	installation := &kotsv1beta1.Installation{
 		Spec: InstallationSpecFromVersionInfo(ctx.VersionInfo),
 	}
-	registryProxyInfo := registry.GetRegistryProxyInfo(ctx.license, installation, ctx.app)
+	registryProxyInfo, err := registry.GetRegistryProxyInfo(ctx.license, installation, ctx.app)
+	if err != nil {
+		// TODO: log
```

Comment:
i don't get this use case, shouldn't this still be an error?

Reply:
updated